Dockerfile

```diff
@@ -1,32 +1,45 @@
 # WebApollo
-# VERSION 2.0
-FROM ubuntu:16.10
-MAINTAINER Eric Rasche <esr@tamu.edu>, Nathan Dunn <nathandunn@lbl.gov>
+# VERSION master
+FROM tomcat:8-jre8
+MAINTAINER Anthony Bretaudeau <anthony.bretaudeau@inra.fr>, Eric Rasche <esr@tamu.edu>, Nathan Dunn <nathandunn@lbl.gov>
 ENV DEBIAN_FRONTEND noninteractive
 
 RUN apt-get -qq update --fix-missing && \
-	apt-get --no-install-recommends -y install \
-	git build-essential maven libpq-dev postgresql-common tomcat8 openjdk-8-jdk nodejs npm wget \
-	postgresql postgresql-client xmlstarlet netcat  \
```

Comment:
you're removing postgres here, in the past you've expressed a desire for a unified image, something users can start without needing to also launch a postgres container. It definitely affects many of the issues mentioned below.